tests/__init__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this header necessary?
I don't see any characters that could cause encoding errors.